webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -500,6 +500,36 @@ public function viewAction(Request $request, int $submitId): Response
                 'after' => 15,
                 'url' => $this->generateUrl('jury_submission', ['submitId' => $submission->getSubmitid()]),
             ];
+        } else {
+            $contestProblem = $submission->getContestProblem();
+            /** @var JudgeTask $judgeTask */
+            $judgeTask = $this->em->getRepository(JudgeTask::class)->findOneBy(['jobid' => $selectedJudging->getJudgingid()]);
+            if ($judgeTask !== null) {
+                $errors = [];
+                $this->maybeGetErrors('Compile config',
+                    $this->dj->getCompileConfig($submission),
+                    $judgeTask->getCompileConfig(),
+                    $errors);
+                $this->maybeGetErrors('Run config',
+                    $this->dj->getRunConfig($contestProblem, $submission),
+                    $judgeTask->getRunConfig(),
+                    $errors);
+                $this->maybeGetErrors('Compare config',
+                    $this->dj->getCompareConfig($contestProblem),
+                    $judgeTask->getCompareConfig(),
+                    $errors);
+                if (!empty($errors)) {
+                    if ($selectedJudging->getValid()) {
+                        $type = 'danger';
+                        $header = "Some parameters have changed since the judging was created, consider rejudging.\n\n";
+                    } else {
+                        $type = 'warning';
+                        $header = "Some parameters have changed since the judging was created, but this judging is no longer valid.\n\n";
```

Comment:
```suggestion
                        $header = "Some parameters have changed since the judging was created, but this judging has been superseded, please verify if that needs a rejudging.\n\n";
```

So probably another wording is better, but I think this is unclear for first time users which are also the people who might change the settings during the contest.